test/visitors/test_cwp_connectivity_visitor.py

```diff
@@ -0,0 +1,32 @@
+from xml.etree.ElementTree import Element
+from bpmncwpverify.core.cwp import Cwp, CwpState, CwpEdge
+from bpmncwpverify.visitors.cwp_connectivity_visitor import CwpConnectivityVisitor
+
+
+def test_cwp_connectivity():
```

Comment:
Let's use the builder interface for creating the `Cwp`? Or is it easier to keep the test directly instantiating the `Cwp` instance as it does here?

Reply:
Yes, that is a great idea.